htdocs/libraries/icms/member/user/Handler.php

```diff
@@ -98,6 +98,24 @@ public function &get($id) {
 		return $user;
 	}
 
+	/**
+	 * retrieve a user from hash_uid
+	 *
+	 * @param string $hash_uid hash_uid of the user
+	 * @return mixed reference to the {@link icms_member_user_Object} object, FALSE if failed
+	 */
+	public function &getHash($hash_uid) {
+		$user = FALSE;
+		$sql = "SELECT * FROM " . $this->db->prefix('users') . " WHERE hash_uid = '" . $hash_uid . "'";
```

Comment:
`hash_uid` should be escaped in case of SQL Injection. `$this->db->quote($hash_uid)` or `$this->db->quoteString($hash_uid)` should be used here